Detectors/TRD/simulation/include/TRDSimulation/SimParam.h

```diff
@@ -111,50 +127,9 @@ class SimParam
   float mInvTRFwid{static_cast<float>(mTRFbin) / (mTRFhi - mTRFlo)}; ///<  Inverse of the bin width of the integrated TRF and x-talk
   float mGasGain{4000.f};                                            ///< Gas gain
 
- private:
-  /// This is a singleton, constructor is private!
-  SimParam();
-  ~SimParam() = default;
-
-  /// Fill the arrays mTRDsmp and mCTsmp for the given gas mixture
-  void sampleTRF();
-
   ClassDefNV(SimParam, 1); // The TRD simulation parameters
```

Comment:
I guess you'll need to increment the version as well

Reply:
Yes that makes sense. In principle though this is only important if we have stored the SimParam in a file somewhere and would like to open it again, right? Because otherwise ROOT complains about a different class layout? I don't think we do have it in a file somewhere, but I will change it nevertheless to be sure.